ppafm/cli/generateElFF.py

```diff
@@ -77,14 +88,18 @@ def main():
 
     if args.tip_dens is not None:
         ###  NO NEED TO RENORMALIZE : fieldFFT already works with density
-        print(">>> loading tip density from ",args.tip_dens,"...")
-        rho_tip, lvec_tip, nDim_tip, head_tip = io.loadXSF( args.tip_dens )
-
+        print(">>> Loading tip density from ",args.tip_dens,"...")
+        if(args.tip_dens.lowercase().endswith("xsf") ):
+            rho_tip, lvec_tip, nDim_tip, head_tip = io.loadXSF( args.tip_dens )
+        elif(args.tip_dens.lowercase().endswith("cube") ):
+            rho_tip, lvec_tip, nDim_tip, head_tip = io.loadCUBE( args.tip_dens )
```

Comment:
Actually, this doesn't work, because the `loadCUBE` function automatically multiplies the array by the conversion factor between Hartree and eV, but here we have an electron density instead of a Hartree potential. I would remove this for now. If we want to support .cube also for electron densities, we need to fix the problem with the factor, which is probably another issue.